tasks.py

```diff
@@ -518,25 +518,30 @@ def test(c, database=None):
     manage(c, 'test', pty=True)
 
 
-@task(pre=[update], help={'dev': 'Set up development enviroment at the end'})
-def setup_test(c, dev=False):
+@task(help={'dev': 'Set up development enviroment at the end'})
+def setup_test(c, ignore_update=False, dev=False, path="inventree-demo-dataset"):
```

Comment:
Question: Why the rename? Was this path already in use?
Suggestion: This name should also be in the .gitignore